src/InputTypes/Credentials.js

```diff
@@ -34,12 +34,35 @@ class CredentialsTooltipItem {
  */
 function fromPassword (password) {
     return {
-        id: GENERATED_ID,
+        autogenerated: true,
         password: password,
-        username: ''
+        username: '',
+        id: ''
```

Comment:
`id` here is redundant. The native layer doesn't use it, their database generates one automatically. I'd remove it to avoid confusion.